remotesigning/remote_signing.go

```diff
@@ -34,41 +34,54 @@ func HandleRemoteSigningWebhook(
 	webhook webhooks.WebhookEvent,
 	seedBytes []byte,
 ) (string, error) {
+	if !validator.ShouldSign(webhook) {
+		return DeclineToSignMessages(client, webhook)
+	}
+
+	response, err := GraphQLResponseForRemoteSigningWebhook(webhook, seedBytes)
+
+	if err != nil {
+		return "", err
+	}
+
+	if response == nil {
+		// No response is required for this event type.
+		return "", nil
+	}
+
+	return HandleSigningResponse(client, response)
+}
+
+func GraphQLResponseForRemoteSigningWebhook(
+	webhook webhooks.WebhookEvent,
+	seedBytes []byte,
+) (SigningResponse, error) {
 	if webhook.EventType != objects.WebhookEventTypeRemoteSigning {
-		return "", errors.New("webhook event is not for remote signing")
```

Comment:
Do we need `validator.ShouldSign` in this method?

Reply:
TBH I don't think this validator interface is needed at all. But I'll add it here as well for consistence purpose.